slack/tool.gpt

```diff
@@ -192,6 +192,21 @@ Credential: ./credential
 
 #!/usr/bin/env node ${GPTSCRIPT_TOOL_DIR}/index.js userContext
 
+---
+Name: Format Message
```

Comment:
Could this be an input filter instead?

Reply:
i have tried input filter and it doesn't work well. https://acorn-io.slack.com/archives/C03FU91U4SX/p1742329698980689